tools/ColorTool/ColorTool/Program.cs

```diff
@@ -208,6 +214,37 @@ static bool SetDefaults(uint[] colorTable)
             return true;
         }
 
+        static bool ExportCurrentAsIni(string outputPath)
```

Comment:
Since there are multiple ways we may want to export, it would be nice to have some interface we could program against (ie `ISchemeExporter` in contrast to `ISchemeParser`) so ini, iterm, etc could be used

Reply:
Indeed, exporting to .itermcolors should be possible, not only to .ini

Reply:
So as to not back up this code review any longer, I'm going to "Won't Fix" this and link it to the new issue #160. 

My goal at this moment is to clear this backlog of pulls. Thanks.